src/binaryen-c.h

```diff
@@ -1180,6 +1180,15 @@ BINARYEN_API void BinaryenSetFunctionTable(BinaryenModuleRef module,
                                            const char** funcNames,
                                            BinaryenIndex numFuncNames,
                                            BinaryenExpressionRef offset);
+BINARYEN_API int BinaryenIsFunctionTableImported(BinaryenModuleRef module);
+BINARYEN_API BinaryenIndex
+BinaryenGetFunctionTableNumSegments(BinaryenModuleRef module);
+BINARYEN_API BinaryenExpressionRef BinaryenGetFunctionTableSegmentOffset(
+  BinaryenModuleRef module, BinaryenIndex id);
+BINARYEN_API BinaryenIndex BinaryenGetFunctionTableSegmentDataLength(
```

Comment:
I think it would be fine to drop `Data` and call this `BinaryenGetFunctionTableSegmentLength` if you want a shorter name.